src/util/regexp.c

```diff
@@ -125,7 +125,7 @@ int git_regexp_search(const git_regexp *r, const char *string, size_t nmatches,
 
 	if ((data = pcre2_match_data_create(nmatches, NULL)) == NULL) {
 		git_error_set_oom();
-		goto out;
+		return GIT_ERROR;
```

Comment:
For consistency, we usually just use `-1`, even though we have `GIT_ERROR`. 

```suggestion
		return -1;
```